cover.out

```diff
@@ -0,0 +1,176 @@
+mode: set
```

Comment:
This file should be `gitignore`-d 🤔

Reply:
It will be, in the other PR.

`git add -u` can be your friend as well 😂

Reply:
:flushed: